index.js

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env node
+
+const core = require("@actions/core");
+const github = require("@actions/github");
+const { Octokit, App } = require("octokit");
+
+async function run() {
+    const githubToken = core.getInput("github_token");
+    const debug = core.getInput("debug") === "true" || false;
```

Comment:
No need to use a fallback here, the value will always be a boolean.
```suggestion
    const debug = core.getInput("debug") === "true";
```